Source/WebKit/WebProcess/WebPage/WebPage.h

```diff
@@ -1903,6 +1904,10 @@ class WebPage final : public API::ObjectImpl<API::Object::Type::BundlePage>, pub
 
     bool isClosed() const { return m_isClosed; }
 
+#if HAVE(AUDIT_TOKEN)
+    void setPresentingApplicationAuditToken(CoreIPCAuditToken&&);
```

Comment:
Why passing a token, it seems only pid is really used?
In the future, we might stick to PageIdentifier to get the token in GPUProcess.

Reply:
My thinking was that the audit token should be the sole source of truth rather than storing PIDs in once place and audit tokens in another. So, that's why I'm storing an audit token in the WebContent process for the various places that want to know the presenting application PID.

I agree though that for PIDToInheritApplicationStateFrom it would be simpler to just pass a PageIdentifier to the GPUProcess.